docs/shared/recipes/nx-release/customize-conventional-commit-types.md

```diff
@@ -0,0 +1,92 @@
+# Customize Conventional Commit Types
+
+Nx Release can defer to the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) standard to automatically determine the next version to release. To enable this behavior for versioning, see [Automatically Version with Conventional Commits](/recipes/nx-release/automatically-version-with-conventional-commits).
+
+This recipe will cover how to customize the types of commits that trigger version bumps, how to customize the version bump for each type, and how to customize the changelog entry for each commit type.
+
+## Conventional Commits Usage within Nx Release
+
+The conventional commits configuration is used in two different places within Nx Release - once in the version step for determining the version bump, and once when generating changelogs.
+
+### Determine the Version Bump
+
+When `release.version.conventionalCommits` is `true` in `nx.json`, Nx Release will use the commit messages since the last release to determine the version bump. It will look at the type of each commit and determine the highest version bump from the following list:
+
+- 'feat' -> minor
```

Comment:
Missing breaking changes for major and a description of how they work (and how users, in the common case, are not expected to configure major bumps to go along with types themselves, and instead should understand how breaking changes are layered on top by us via `!` or `BREAKING CHANGE` - as per the conventional commits specification)